### DISCUSSION TITLE
合集页面在侧栏添加单集视频BV号的显示

### DISCUSSION THREAD

**Discussion by hetan697** [the1812/Bilibili-Evolved#1604] [功能建议 / Ideas]
首先感谢脚本开发者，本人非常喜欢这个脚本提供的三倍变速、顶栏自定义、跳过鸣谢等功能。我想提的建议是，**能不能考虑在合集页面加个当前视频的BV号显示，或者是添加一个“跳转至原视频”的按钮**，放在普通视频界面的av/BV号的展示位置就行😂

如果脚本开发者仍有余力，或许可以考虑合集页面（比如`https://www.bilibili.com/medialist/play/ml1188354743/BV1mb4y1X7Bw`这样的）优化下展示方式。现在这样视频播放器在上、视频标题和UP主信息在下很不习惯😥。能不能**在不对网页重定向的基础上**弄成和普通视频播放的页面差不多的。

当然，如果工作量很大或者说根本没法做就算了，这事本来就是批站的睿智操作造成的。

不管这个诉求怎么处理结果怎么样，都感谢开发者大大开发出的脚本，谢谢！！

（再啰嗦一句：改页面这需求太扯了）

**Comment by Geeyun-JY3** [comment]
合集页面点击下方视频标题就能跳转至原视频

**Reply by Geeyun-JY3** [reply]
还有啊，你上面那个网址最后就是你要的当前视频的 BV 号

**Comment by Geeyun-JY3** [comment]
> 弄成和普通视频播放的页面差不多的。

好的你这需求官方在最近给你实现了。现在稍后再看和收藏播放列表页面和新加入的 UP 主视频合集页面基本一致。
